charts/agent/README.md

```diff
@@ -169,11 +169,11 @@ Chart to install K8s collection stack based on Observe Agent
 | cluster.metrics.pod.action | string | `"keep"` |  |
 | cluster.metrics.pod.enabled | bool | `false` |  |
 | cluster.metrics.pod.interval | string | `"10s"` |  |
-| cluster.metrics.pod.metric_drop_regex | string | `".*bucket"` |  |
-| cluster.metrics.pod.metric_keep_regex | string | `"(.*)"` |  |
-| cluster.metrics.pod.namespace_drop_regex | string | `"(.*istio.*|.*ingress.*|kube-system|observe)"` |  |
-| cluster.metrics.pod.namespace_keep_regex | string | `"(.*)"` |  |
-| cluster.metrics.pod.port_keep_regex | string | `".*metrics"` |  |
+| cluster.metrics.pod.metricDropRegex | string | `".*bucket"` |  |
```

Comment:
re: these being nested under cluster, we don't need the top level groups to match exactly with the services we're actually deploying since these values just get used downstream in the actual configs as flags. I think its better to organize top level by use case/category of data in which case `application` makes more sense as a standalone. In the future if we want to have some configurations around traces at this level too it would make sense under `application` moreso than cluster

Reply:
names look good to me though, i do kind of want some sort of comment explaining how the drop and keep interact. what happens if i define rules that collide? like i want to keep everything but then drop some things? drop takes precedent I assume? would love to have that written explicitly

Reply:
done